tests/test_runner/test_hooks.py

```diff
@@ -355,6 +355,97 @@ def test_cosine_runner_hook(multi_optimziers):
     hook.writer.add_scalars.assert_has_calls(calls, any_order=True)
 
 
+@pytest.mark.parametrize('multi_optimziers', (True, False))
+def test_flat_cosine_runner_hook(multi_optimziers):
```

Comment:
maybe we should add unittest like https://github.com/open-mmlab/mmcv/blob/bf2c9fa8d21e4af903769b3d7aca20452e4b4669/tests/test_runner/test_hooks.py#L363